storage-node/src/commands/server.ts

```diff
@@ -79,6 +79,18 @@ export default class Server extends ApiCommandBase {
 Log level could be set using the ELASTIC_LOG_LEVEL enviroment variable.
 Supported values: warn, error, debug, info. Default:debug`,
     }),
+    elasticSearchIndex: flags.string({
+      required: false,
+      description: 'Elasticsearch index name.',
+    }),
+    elasticSearchUser: flags.string({
+      required: false,
+      description: 'Elasticsearch user for basic authentication.',
+    }),
+    elasticSearchPassword: flags.string({
```

Comment:
generally I don't like passing passwords as command line arguments, it is less secure (password would be available to see in shell history, process information, deployment scripts (although there are ways to secure secrets in pulumi deployment scripts for example).
But given that it is just for elastic search authentication I think its fine.

Reply:
We easily add add the `env` option to set a default value from the environment.
It might also make sense to use the `dependsOn` property instead of `required` on both `elasticSearchPassword` and `elasticSearchUser` to make them depend on each other.

see https://oclif.io/docs/flags

Reply:
Done in 139c5b700a99fc54e26f80a177fb9711db5d49f7